test/fixtures/dialects/tsql/tablesample.sql

```diff
@@ -0,0 +1,11 @@
+SELECT *
+FROM Sales.Customer TABLESAMPLE SYSTEM (10 PERCENT);
+
+SELECT *
+FROM Sales.Customer TABLESAMPLE (10 ROWS);
+
+SELECT *
+FROM Sales.Customer TABLESAMPLE (10)
```

Comment:
why does it work without ending semicolon? t-sql thing?

Reply:
Technically not required with T-SQL most people still put them in though to align with other SQL formats.https://learn.microsoft.com/en-us/sql/t-sql/language-elements/transact-sql-syntax-conventions-transact-sql?redirectedfrom=MSDN&view=sql-server-ver16

Will update the test case to be consistent though